deployments/examples/cs3_users_ocis/docker-compose.yaml

```diff
@@ -113,11 +113,14 @@ services:
     command: --copy-service --loglevel debug
     environment:
       LDAP_TLS_VERIFY_CLIENT: never
-      LDAP_DOMAIN: owncloud.test
+      LDAP_DOMAIN: owncloud.com
       LDAP_ORGANISATION: ownCloud
       LDAP_ADMIN_PASSWORD: ${LDAP_ADMIN_PASSWORD:-admin}
       LDAP_RFC2307BIS_SCHEMA: "true"
       LDAP_REMOVE_CONFIG_AFTER_SETUP: "false"
+    ports:
+      - "389:389"
+      - "636:636"
```

Comment:
Can we only map them to the localhost? If this example is installed on a server it might be exposed to the internet.
This would make it usable for local testing:
```
ports:
   - "127.0.0.1:389:389"
   - "127.0.0.1:636:636"
```

Reply:
:+1: